articles/quickstart/native/android/05-linking-accounts.md

```diff
@@ -116,33 +139,20 @@ The process to link accounts requires the logged-in user's ID and the id tokens
 // app/src/main/java/com/auth0/samples/activities/LoginActivity.java
 
 private void performLink(final String secondaryIdToken) {
-  credentialsManager.getCredentials(new BaseCallback<Credentials, CredentialsManagerException>() {
-      @Override
-      public void onSuccess(Credentials credentials) {
-          UsersAPIClient client = new UsersAPIClient(auth0, credentials.getIdToken());
-          String primaryUserId = getIntent().getExtras().getString(Constants.PRIMARY_USER_ID);
-          client.link(primaryUserId, secondaryIdToken)
-                  .start(new BaseCallback<List<UserIdentity>, ManagementException>() {
-                      @Override
-                      public void onSuccess(List<UserIdentity> userIdentities) {
-                          //Accounts linked
-                      }
-
-                      @Override
-                      public void onFailure(ManagementException error) {
-                          //Linking failed
-                      }
-                  });
-
-      }
-
-      @Override
-      public void onFailure(CredentialsManagerException error) {
-          //Credentials probably expired. Remove them and login again
-          credentialsManager.clearCredentials();
-          finish();
-      }
-  });
+    UsersAPIClient client = new UsersAPIClient(auth0, getIntent().getExtras().getString(KEY_ID_TOKEN));
+    String primaryUserId = getIntent().getExtras().getString(KEY_PRIMARY_USER_ID);
```

Comment:
The Management API endpoints now receive an access_token as a bearer and not an id_token, how is this working in the Android SDK?

Reply:
Both our native SDKs use the same endpoints with an id_token, and they both work ATM. No need to hold this PR because of something that hasn't been announced yet. 👍